### ISSUE TITLE
Screenreader Issue - triangles example alt text

### ISSUE THREAD

**Issue by James-Green** [w3c/wai-quick-start#287]
In Designing, the triangle graphics don’t have adequate alt text in my opinio.  They are currently “inaccessible triangle question” and “accessible triangle question” when I think they should be communicating the point we are making, e.g., “triangles of different types and colors” and “triangles of different types and colors, each labeled with a number”
![screen shot 2015-09-30 at 9 56 21 pm](https://cloud.githubusercontent.com/assets/13589074/10211901/ddbea534-67bd-11e5-80ee-d07b14d1d0d8.png)

**Comment by iadawn** [comment]
For this one I have maded the bad one bad and the good one better.

For the bad one, the SVG has `role='img'` and a rubbish alternative text.

For the good one, the SVG does not have `role='img'` and does not have alternative text. The SVG does have `<text>` content which explains the triangles and makes it possible to actually answer the question.

**Comment by James-Green** [comment]
nice!
